xcodeproj/internal/templates/xcodeproj.bazelrc

```diff
@@ -67,6 +67,10 @@ build:rules_xcodeproj --show_result=0
 build:rules_xcodeproj --noworker_sandboxing
 build:rules_xcodeproj --spawn_strategy=remote,worker,local
 
+# Use param files.
```

Comment:
Can you move this up to line 59 (just after the last `--features`. Also add a TODO to remove once apple_support 1.6.0 is our minimum supported version.

Reply:
Done.